main/src/library/ChannelImpl.flix

```diff
@@ -0,0 +1,379 @@
+//
+// Copyright 2018 Simon Meldahl Schmidt
+// Copyright 2018-2021 Jonathan Lindegaard Starup
+// Copyright 2021 Justin Fargnoli
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace ChannelImpl {
+
+    ///
+    /// The tuple corresponds to (
+    ///   `id`,
```

Comment:
a unique identifier used for what? (to sort the channel?)

Reply:
yes, added comment